.github/ISSUE_TEMPLATE/1-bugreport.yml

```diff
@@ -0,0 +1,55 @@
+name: "Bug Report"
+description: "File a bug report"
+title: "[Bug]: "
+labels: ["bug"]
```

Comment:
Is there a need to do both (i.e., have `[Bug]` prefix and set the label to `bug`)? I think just setting the `bug` label is sufficient.

Reply:
removed